ios-trivia-game/ViewControllers/GameOptionsViewController.swift

```diff
@@ -10,17 +10,62 @@ import UIKit
 
 class GameOptionsViewController: UIViewController {
 
+    @IBOutlet weak var categoryPicker: UIPickerView!
+    @IBOutlet weak var numOfQuestionsPicker: UIPickerView!
+    
+    var categoryPickerData: [String] = [String]()
+    var numOfQuestionsPickerData: [String] = [String]()
+    
+    var numOfQuestions: Int?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
         // Do any additional setup after loading the view.
+        getCategories()
+    }
+    
+    func getCategories() {
+        JServiceClient.instance.categories(count: 10, offset: 10, success: {(result: Any?) in
```

Comment:
I was thinking that ideally we can let users to choose how many categories they want, and generate offset randomly, so that it can give different categories every time, but I just hard-coded for now. Do you want to use hardcoded array? or want to tweak this numbers?